<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Benutzerlevel</class>
 <widget class="QDialog" name="Benutzerlevel">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minefiled - User definated level</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width</string>
       </property>
       <property name="buddy">
        <cstring>breite_box</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="breite_box">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height</string>
       </property>
       <property name="buddy">
        <cstring>hoehe_box</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="hoehe_box">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mines</string>
       </property>
       <property name="buddy">
        <cstring>mienen_box</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="mienen_box">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="abbrechen_button">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>breite_box</tabstop>
  <tabstop>hoehe_box</tabstop>
  <tabstop>mienen_box</tabstop>
  <tabstop>abbrechen_button</tabstop>
  <tabstop>ok_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>abbrechen_button</sender>
   <signal>clicked()</signal>
   <receiver>Benutzerlevel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
